src/layouts/NavbarWrapper.tsx

```diff
@@ -1,11 +1,27 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Navbar } from '@/utils/navbar'
 
-export default function NavbarWrapper( { children } ) {
+
+export default function NavbarWrapper({ children }) {
+
+  const [isEaster, toggleEaster] = useState(false);
+
+  const changeTheme = () => {
+    toggleEaster(isEaster => !isEaster)
+  }
+
+  let propsObj = {
+    changeTheme: changeTheme, 
+    isEaster: isEaster
+  }
+
   return (
     <>
-        <Navbar />
+      <div className={`border-4 ${isEaster? "easter" : ""} border-skin-muted bg-skin-primary h-screen`}>
```

Comment:
Awesome! So everything within this wrapper is default/root them or easter theme? Would this make sense to put a higher level up (app.tsx)? Unless there's a separate wrapper in the future, this is the highest up, I suppose.

My gut instinct is that the theme being passed in NavbarWrapper.tsx is a little confusing (i.e. not the first place one might search for it), but I suppose the counterargument is that we have to pass changeTheme up another function. Curious as to the reasoning if there's anything I'm missing 😊

Reply:
On this note, I'm (finally 😅) getting Auth0 planned out and one easy option is to use Layout which preserves the its state on every page:
https://nextjs.org/docs/basic-features/layouts

I'm going to try and get my understanding more settled by the time we chat tomorrow so hopefully we can make an informed decision. At this point I'm really just shooting from the hip... :P

Reply:
I think it makes sense here in terms of structure, but navbar wrapper should definitely be renamed, since it's a general wrapper for all components. It's a layout, so it could be called something like "page Layout," but I think it adding the class to something that's explicitly about the components rather than app keeps it more understandable. But also... I don't have any idea what best practice is here, that's just my vibe lol

Reply:
That makes sense! I like the idea of refactoring this to be multi-purpose and utilize Layout. We can do that in a future commit/PR but if you have a vision for it at the moment feel free to update and push a new commit up ^^